tests/draft2019-06/refRemote.json

```diff
@@ -139,7 +135,7 @@
             "$id": "http://localhost:1234/object",
             "type": "object",
             "properties": {
-                "name": {"$ref": "name.json#/definitions/orNull"}
+                "name": {"$ref": "name.json#/$defs/orNull"}
```

Comment:
We may need a new */remotes/name.json*:

```
{
    "definitions": {
        "orNull": {
            "anyOf": [
                {"type": "null"},
                {"$ref": "#"}
            ]
        }
    },
    "type": "string"
}
```

It has a `definitions` that 3 of the tests try to reference into.

- refRemote
  - root in remote ref
    - null is valid
    - object is invalid
    - string is valid

This change updates the `definitions` to a `$defs` in the `$ref`s causing the tests to (rightly) fail.

Reply:
Don't just change _/remotes/name.json_, though.  We still need that for the other drafts.  (I suppose you could _add_ `$defs` with a copy of what's in `definitions`... ¯\\\_(ツ)\_\/¯

Reply:
Ech, yes that sounds annoying to fix...

Reply:
I actually tried to add `$defs` to *name.json* locally, but my implementation threw a fit because it couldn't determine which draft the schema was.

I have logic which analyzes the keywords to determine which drafts the schema conforms to if it's not specified explicitly by the `$schema` keyword. Since I don't list `definitions` as draft-8-compatible, having both it and `$defs` in the same schema gives it a sad.

I understand that `definitions` is included for backward compatibility, but I'm trying to dissuade people from using it by producing an error when it is used in a draft-8 schema.

In the end, I ended up creating a *name-draft-8* file to get it passing locally.